CodeForPhilly/vacant-lots-proj
# 304 label map regions

For https://github.com/CodeForPhilly/vacant-lots-proj/issues/304

### Testing
- Go to the maps page
- Turn on screenreader
- Open landmarks shortcut
- You should see maps/ results sidepanel/controls listed among the other landmarks
